<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: #646464;
	color: #eee;
}
QLabel
{
	background-color: transparent;
	color: #000;
}

QMenuBar 
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(185, 185, 185, 255),stop:1 rgba(183, 183, 183, 255));
	border: 1px solid #4f4f4f;
	color: #000;
}

QMenuBar::item 
{
	background-color: transparent;

}

QMenuBar::item:selected 
{
	background-color: #d1d4da;
	color: #000;
}
QMenuBar::item:pressed 
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(89, 89, 89, 255),stop:1 rgba(66, 66, 66, 255));
	border: 1px solid #000;
	color: #fff;
}

QMenu
{
    background-color: #fff;
    border: 1px solid #222;
    padding: 4px;
	color: #000;
}
QMenu::item
{
    background-color: transparent;
    padding: 2px 20px 2px 20px;
}
QMenu::item:disabled
{
    color: #555;
    background-color: transparent;
    padding: 2px 20px 2px 20px;

}
QMenu::item:selected
{
    background-color: #91c9f7;
    color: #000;
}

QToolBar
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(185, 185, 185, 255),stop:1 rgba(183, 183, 183, 255));
	border-top: none;
	border-bottom: 1px solid #343434;
	border-left: 1px solid #343434;
	border-right: 1px solid #343434;

}

QToolButton 
{
	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(232, 232, 232, 255),stop:1 rgba(190, 190, 190, 255));
	color: #eee;
	border: 1px solid #4f4f4f;
	border-radius: 4px;
	padding: 2px;
	margin-left: 5px
}


QToolButton:hover
{
	background-color: #8b8b8b;
	
}


QToolButton:pressed
{
	background-color: #4f4f4f;
	border: 1px solid gray;

}


QToolButton:checked
{
	background-color: #4f4f4f;
	border: 1px solid gray;
}

QLineEdit
{
	background-color: #fff;
	color : #000;
	border: 1px solid #343434;
	border-radius : 2px;
	padding: 2px;

}

/*-----QPushButton-----*/
QPushButton
{
	background-color: #3de6af;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::hover
{
	background-color: #3dffaf;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::pressed
{
	background-color: #3dc8af;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}
/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #082336;
	color: #3e93ff;
	border-style: solid;
	border-width: 2px;
	border-color: #1e4b6d;
	padding: 10px;

}


QLineEdit::hover
{
	background-color: #082336;
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;
}
QLineEdit::focus
{
	color: #3e93ca;
	border-style: solid;
	border-width: 2px;
	border-color: #01dcff;
	padding: 10px;
}
QCheckBox
{
	background-color: transparent;
	color: #717e87;
	border-color: #000000;
}
QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #00111d;
    border: 1px solid #2aaaa8;
    width: 12px;
    height: 12px;
}
QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;); /*To replace*/
	background-color: #1f2b2b;
    border: 1px solid #2aaaa8;
}
QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #2aaaa8;
}
QCheckBox::disabled
{
	color: #656565;
}

QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}
TextEdit {
  list-style-type: none;
  margin: 0;
  padding: 0;
  overflow: hidden;
  background-color: #333333;
  float: left;
}
TextEdit::hover {
  background-color: #111111;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="text">
         <string>Словарь</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Введите слова или фразу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Найти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">TextEdit [text]{
  list-style-type: none;
  margin: 0;
  padding: 0;
  overflow: hidden;
  background-color: #333333;
}
li {
  float: left;
}
TextEdit:hover {
  background-color: #111111;
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Arial'; font-size:10px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="create"/>
    <addaction name="open_tr"/>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>О &amp;нас</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="create"/>
   <addaction name="open_tr"/>
   <addaction name="quit"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionaddWord"/>
   <addaction name="actionremoveWord"/>
   <addaction name="actionWhatsUp"/>
   <addaction name="actionTelegramm"/>
   <addaction name="actionrat_1"/>
   <addaction name="actionrat_2"/>
   <addaction name="actionrat_3"/>
  </widget>
  <action name="create">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/create_tr.ico</normaloff>:/img/img/create_tr.ico</iconset>
   </property>
   <property name="text">
    <string>Создать Cловарь</string>
   </property>
  </action>
  <action name="quit">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/power.ico</normaloff>:/img/img/power.ico</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="whatsThis">
    <string>Выход с программы</string>
   </property>
   <property name="shortcut">
    <string>Alt+Ч</string>
   </property>
  </action>
  <action name="open_tr">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/open_tr.ico</normaloff>:/img/img/open_tr.ico</iconset>
   </property>
   <property name="text">
    <string>Открыт Словарь</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>О программы</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Раработчик</string>
   </property>
  </action>
  <action name="actionaddWord">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/add_word.ico</normaloff>:/img/img/add_word.ico</iconset>
   </property>
   <property name="text">
    <string>addWord</string>
   </property>
   <property name="toolTip">
    <string>Добавление слова</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionremoveWord">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/remove_word.ico</normaloff>:/img/img/remove_word.ico</iconset>
   </property>
   <property name="text">
    <string>removeWord</string>
   </property>
   <property name="toolTip">
    <string>Удаление слова</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionWhatsUp">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/whatsapp.ico</normaloff>:/img/img/whatsapp.ico</iconset>
   </property>
   <property name="text">
    <string>WhatsUp</string>
   </property>
   <property name="toolTip">
    <string>Перейти в группу</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionTelegramm">
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/telegram.ico</normaloff>:/img/img/telegram.ico</iconset>
   </property>
   <property name="text">
    <string>Telegramm</string>
   </property>
   <property name="toolTip">
    <string>Перейти в канал</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionrat_1">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/star.ico</normaloff>
     <activeoff>:/img/img/start_sl.ico</activeoff>
     <selectedoff>:/img/img/start_sl.ico</selectedoff>:/img/img/star.ico</iconset>
   </property>
   <property name="text">
    <string>rat_1</string>
   </property>
   <property name="toolTip">
    <string>Рейтинг</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionrat_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/star.ico</normaloff>
     <activeoff>:/img/img/start_sl.ico</activeoff>
     <selectedoff>:/img/img/start_sl.ico</selectedoff>:/img/img/star.ico</iconset>
   </property>
   <property name="text">
    <string>rat_2</string>
   </property>
   <property name="toolTip">
    <string>Рейтинг</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionrat_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresource.qrc">
     <normaloff>:/img/img/star.ico</normaloff>
     <activeoff>:/img/img/start_sl.ico</activeoff>:/img/img/star.ico</iconset>
   </property>
   <property name="text">
    <string>rat_3</string>
   </property>
   <property name="toolTip">
    <string>Рейтинг</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myresource.qrc"/>
 </resources>
 <connections/>
</ui>
